build.gradle

```diff
@@ -1,6 +1,9 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = '1.7.10'
+    ext {
+        kotlin_version = '1.7.0'
```

Comment:
Is there some reason `kotlin_version`'s downgrade?

Reply:
- compose ```v1.2.0``` -> kotlin ```1.7.0``` required
- compose ```v1.2.1``` -> kotlin ```1.7.10``` required

When I tried compose ```v.1.2.1``` which is the latest version, but it fails to build euphony, so I downgraded version of compose and kotlin 😢 
Now, I'm testing again whether compose ```v1.2.1```. If it is working, I will upgrade the version and let you know :)